packages/validator/src/services/validatorStore.ts

```diff
@@ -79,6 +80,8 @@ type ValidatorData = {
  */
 export class ValidatorStore {
   private readonly validators = new Map<PubkeyHex, ValidatorData>();
+  private readonly validatorRegistrationCache = new ValidatorRegistrationCache();
```

Comment:
Can you include a jsdoc comment here or on the class?

Reply:
done!

Reply:
Why not add this inside `ValidatorData` so it gets automatically pruned when removing validators?

Reply:
moved